### DISCUSSION TITLE
Patient files output in .ndjson format?

### DISCUSSION THREAD

**Discussion by ToInf1n1tyAndBeyond** [synthetichealth/synthea#1329] [Q&A] [answered]
Can non-bulk output be generated in .ndjson format?

**Comment by awatson1978** [comment, marked as answer]
Not yet, but I’m currently working on implementing this feature!  Stay tuned.  ETA is within a couple of weeks, hopefully!

**Reply by ToInf1n1tyAndBeyond** [reply]
Thank you! Good to know. Is there some way you can let me know when this feature is ready?

**Reply by awatson1978** [reply]
I'll try to comment on here.  Or you can follow issue #1325 and attached pull requests.
